File: beecrowd/1051.py
renda = float(input())
taxa2 = taxa3 = taxa45 = 0
if renda > 4500:
    taxa2 = 80
    taxa3 = 270
    taxa45 = (renda - 4500) * (7 / 25)
elif 3000 < renda <= 4500:
    taxa2 = 80
    taxa3 = (renda - 3000) * (9 / 50)
elif 2000 < renda <= 3000:
    taxa2 = (renda - 2000) * (2 / 25)
imposto = taxa2 + taxa3 + taxa45
if imposto == 0:
    print('Isento')
else:
    print(f'R$ {imposto:.2f}')

File: beecrowd/1051.c
#include <stdio.h>

int main() {

    float salario, imposto;

    scanf("%f", &salario);

    if(salario <= 2000) {
        printf("Isento\n");
    }else if (salario <= 3000) {
        imposto = (((salario - 2000) * 8) / 100);
        printf("R$ %.2f\n", imposto);
    }else if(salario <= 4500) {
        imposto = ((((salario - 3000) * 18) / 100) + 80);
        printf("R$ %.2f\n", imposto);
    }else {
        imposto = ((((salario - 4500) * 28) / 100) + 350);
        printf("R$ %.2f\n", imposto);
    }
    
    return 0;
}